### ISSUE TITLE
NPE on text highligh

### ISSUE THREAD

**Issue by GoogleCodeExporter** [aymanhs/jsyntaxpane#34]
```
What steps will reproduce the problem?
1. Press Ctrl-F to open the find dialog
2. Click on the Highlight button

What is the expected output? What do you see instead?
Expected result: nothing happens

```

Original issue reported on code.google.com by `fcolasuo...@gmail.com` on 31 Oct 2008 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
Thansk for the report.  Being fixed, and will be released with next version.
```

Original comment by `ayman.al...@gmail.com` on 2 Nov 2008 at 5:20
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ayman.al...@gmail.com` on 2 Nov 2008 at 5:21
- Added labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 0.9.3
```

Original comment by `ayman.al...@gmail.com` on 10 Nov 2008 at 5:22
- Changed state: **Fixed**
